src/libslic3r/GCodeWriter.cpp

```diff
@@ -61,11 +61,9 @@ std::string GCodeWriter::preamble()
         FLAVOR_IS(gcfSmoothie) ||
         FLAVOR_IS(gcfKlipper))
     {
-        if (RELATIVE_E_AXIS) {
+        if (this->config.use_relative_e_distances) {
             gcode << "M83 ; only support relative e\n";
```

Comment:
After this pull request this is wrong statement.

Reply:
good catch.